docs/compatibility.md

```diff
@@ -52,7 +52,7 @@ We don't support `response.is_ok` since the naming is ambiguous there, and might
 
 ## Client instances
 
-The HTTPX equivalent of `requests.Session` is `httpx.Client`.
+The HTTPX equivalent of `requests.Session` is `httpx.AsyncClient`.
```

Comment:
Since requests is sync-only... will this be changed _back_ once sync is reintroduced?

Reply:
Yes, pending some final decisions on exact naming for the sync vs async variants.